### PR TITLE
new package: leocad

### PR DESCRIPTION
**Pull request by drvirgilio**

### CODE REVIEW

**Comment by lethalman** [comment]
http://googledrive.com/host/0B4QpN1vpGh_sUWJZSkVVaEtzcW8/LeoCAD-Source-0.80.3.tgz doesn't exist.

**Comment by echo-oddly** [comment]
I fixed it by grabbing the source directly from the svn repo and I removed the parts library since it is optional and changes often.

**Comment by robberer** [comment]
Nice, builds fine on my machine.

File: pkgs/applications/graphics/leocad/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+stdenv.mkDerivation rec {
+  name = "leocad-${version}";
+  version = "0.81";
+
+  src = fetchsvn {
+    url = "http://svn.leocad.org/tags/leocad-0.81";
+    sha256 = "1190gb437ls51hhfiwa79fq131026kywpy3j3k4fkdgfr8a9v3q8";
+  };
+
+  buildInputs = [ makeWrapper qt4 zlib ];
```
Comment (robberer) [inline pkgs/applications/graphics/leocad/default.nix]: Why makeWrapper in buildInputs ? Seems to be unused.

File: pkgs/applications/graphics/leocad/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*
+To use aditional parts libraries
+set the variable LEOCAD_LIB=/path/to/libs/ or use option -l /path/to/libs/
+*/
+
+{ stdenv, fetchurl, fetchsvn, makeWrapper, qt4, zlib }:
```
Comment (robberer) [inline pkgs/applications/graphics/leocad/default.nix]: fetchurl is unused, too

**Comment by robberer** [comment]
Very funny apllication. Years ago, around 2004, I modelled a Snowspeeder myself. Took me weeks to get all single pieces together. This experiment with Leocad and Blender took only minutes for nice result.

Thanks.

http://imgur.com/cV6Ir8n

**Comment by echo-oddly** [comment]
@robberer I forgot to remove those dependencies when making an edit. Thanks for catching that.
Those renders look good. I'm glad it works well for you!

**Comment by robberer** [comment]
Yeah, if no one is nitpicking on line 13, not using ${name} then this should be merged.

**Comment by echo-oddly** [comment]
I fixed that nitpick. Ready for merge.

**Comment by lethalman** [comment]
Built locally, thanks.

**Pull request merged**